addresses.go

```diff
@@ -5,6 +5,6 @@ type Addresses struct {
 	Swarm      []string // addresses for the swarm to listen on
 	Announce   []string // swarm addresses to announce to the network
 	NoAnnounce []string // swarm addresses not to announce to the network
-	API        Strings  // address for the local API (RPC)
-	Gateway    Strings  // address to listen on for IPFS HTTP object gateway
+	API        []string  // addresses for the local API (RPC)
+	Gateway    []string  // addresses to listen on for IPFS HTTP object gateway
```

Comment:
Why are you changing these types?

Reply:
Well, Swarm, Announce and NoAnnounce are arrays, because they hold multiple addresses.

API and Gateway now need to hold multiple addresses too. So they need to be an array.

Or is there's something wrong with my assumption?

Reply:
We created the Strings type to allow either a single string or an array of strings. Changing this to a simple array would be a backwards incompatible change.